wcag20/sources/techniques/general/G85.xml

```diff
@@ -45,20 +45,20 @@
       <procedure>
       <olist>
         <item>
-          <p>要求されているフォーマット又は値ではない入力を故意に行う。</p>
+          <p>フォームに記入し、要求されているフォーマット又は値以外の失敗する入力をを故意に行う。</p>
         </item>
         <item>
-          <p>エラーになった項目を特定して、無効な入力の内容及びその修正方法に関する情報を提供する説明文がテキストで提供される。</p>
+          <p>エラーになった項目を特定して、無効な入力の種類及びその修正方法に関する情報を提供する説明がテキストで提供されていることを確認する。</p>
         </item>
         <item>
-          <p>そのデータが再表示されることが適切ではない、セキュリティに関連するフィールド (例: パスワード) にない限り、利用者によって入力済みのデータが再表示される。</p>
+          <p>再表示のためにデータを保持することが不適切なセキュリティに関連するフィールドにデータ (例: パスワード) がない限り、利用者によって入力済みの他のデータが再表示されていることを確認する。</p>
```

Comment:
>Check that other data previously entered by the user is re-displayed, unless the data is in a security related field where it would be inappropriate to retain the data for re-display (e.g. password).

括弧書きの位置が微妙な感じがします。
「再表示のためにデータを保持することが不適切なセキュリティに関連するフィールド(例: パスワード) にデータがない限り、」
とするほうが良いかもしれません。